### PR TITLE
add make install command

### PR DESCRIPTION
**Pull request by michelleN**
+ for cargo install --path . --locked
+ helps me speed up local dev experience

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,10 @@ endif
 build:
 	cargo build --release
 
+.PHONY: install
+install:
+	cargo install --path .
```
Comment (itowlson) [inline Makefile:32]: Forgive the persnicketing, but `make build` does a release build, while this does a debug build - should they be consistent?

Comment (itowlson) [inline Makefile:32]: ooh might be worth specifying `--locked` as well, otherwise it could pick up random versions of dependencies

Comment (fibonacci1729) [reply Makefile:32]: I believe `cargo install` uses the release profile by default (opposite to `build` which uses `dev`, i.e. `--debug`)

**Review by fibonacci1729 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (itowlson) [reply Makefile:32]: Doh, you're right @fibonacci1729.  Apologies!

Comment (michelleN) [reply Makefile]: done! thanks

**Pull request merged**